### ISSUE TITLE
Container Fails On Windows 11 -isolation Process

### ISSUE THREAD

**Issue by kevindstanley1988** [microsoft/navcontainerhelper#2132]
**Container Fails On Windows 11 -isolation Process on bccontainerhelper v2.0.16**

Artifacturl: https://bcartifacts.azureedge.net/onprem/16.3.14085.14238/na
Windows 11 Pro: 21H2, build 22000.258
Docker Version: 20.10.8, build 3967b7d
Docker Desktop Version: 4.1.1 (69879)
BCContainerHelper Version: 2.0.16

**Scripts used to create container and cause the issue**
```
$licenseFile = 'C:\workpath\license\license 160.flf'
$containerName = 'cronus16'
$artifactURL = 'https://bcartifacts.azureedge.net/onprem/16.3.14085.14238/na'
$containerCredential = Get-Credential -UserName "user1" -Message 'Business Central Credential'

New-BcContainer $containerName -accept_eula -accept_outdated -licenseFile $licenseFile `
            -shortcuts None -includeAL -updateHosts -auth NavUserPassword -memoryLimit "12G" `
            -artifactUrl $artifactURL -Credential $containerCredential -isolation "process" 
```

**Full output of scripts**
```
Windows PowerShell credential request.
Business Central Credential
Password for user user1: ********
BcContainerHelper is version 2.0.16
BcContainerHelper is running as administrator
Host is Microsoft Windows 11 Pro - Unknown/Insider build
Docker Client Version is 20.10.8
Docker Server Version is 20.10.8
Fetching all docker images
Fetching all docker volumes
Using image mcr.microsoft.com/businesscentral:10.0.20348.169
Creating Container cronus16
Style: onprem
Multitenant: No
Version: 16.3.14085.14238
Platform: 16.0.14073.14195
Generic Tag: 1.0.1.7
Container OS Version: 10.0.20348.169 (ltsc2022)
Host OS Version: 10.0.22000.258 (Unknown/Insider build)
WARNING: Host OS and Base Image Container OS doesn't match and process isolation is specified. If you encounter issues, you could try to specify -isolation hyperv
Using process isolation
Using locale en-US
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Using license file C:\workpath\license\license 160.flf
Files in C:\ProgramData\BcContainerHelper\Extensions\cronus16\my:
- AdditionalOutput.ps1
- license.flf
- MainLoop.ps1
- SetupVariables.ps1
- updatehosts.ps1
Creating container cronus16 from image mcr.microsoft.com/businesscentral:10.0.20348.169
ecb27d0d509d34f332a0b558b2805c843ec83920d2cf593147bddc134a19e5cc
Waiting for container cronus16 to be ready
Using artifactUrl https://bcartifacts.azureedge.net/onprem/16.3.14085.14238/na
Using installer from C:\Run\150-new
Installing Business Central
Installing from artifacts
Starting Local SQL Server
Starting Internet Information Server
Copying Service Tier Files
c:\dl\onprem\16.3.14085.14238\platform\ServiceTier\Program Files
c:\dl\onprem\16.3.14085.14238\platform\ServiceTier\System64Folder
Copying PowerShell Scripts
c:\dl\onprem\16.3.14085.14238\platform\WindowsPowerShellScripts\Cloud\NAVAdministration
c:\dl\onprem\16.3.14085.14238\platform\WindowsPowerShellScripts\WebSearch
Copying dependencies
Copying ReportBuilder
Importing PowerShell Modules
Determining Database Collation from c:\dl\onprem\16.3.14085.14238\na\database\Demo Database NAV (16-0).bak
Changing Database Server Collation to Latin1_General_100_CI_AS
Restoring CRONUS Demo Database
Setting CompatibilityLevel for CRONUS on localhost\SQLEXPRESS
Modifying Business Central Service Tier Config File for Docker
Creating Business Central Service Tier
Installing SIP crypto provider: 'C:\Windows\System32\NavSip.dll'
Copying Web Client Files
c:\dl\onprem\16.3.14085.14238\platform\WebClient\Microsoft Dynamics NAV
Copying Client Files
c:\dl\onprem\16.3.14085.14238\platform\LegacyDlls\program files\Microsoft Dynamics NAV
c:\dl\onprem\16.3.14085.14238\platform\LegacyDlls\program files\Microsoft Dynamics NAV
c:\dl\onprem\16.3.14085.14238\platform\LegacyDlls\systemFolder
Copying ModernDev Files
c:\dl\onprem\16.3.14085.14238\platform
c:\dl\onprem\16.3.14085.14238\platform\ModernDev\program files\Microsoft Dynamics NAV
Copying additional files
Copying ConfigurationPackages
C:\dl\onprem\16.3.14085.14238\na\ConfigurationPackages
Copying Test Assemblies
C:\dl\onprem\16.3.14085.14238\platform\Test Assemblies
Copying Applications
C:\dl\onprem\16.3.14085.14238\na\Applications
Starting Business Central Service Tier
Importing license file
Stopping Business Central Service Tier
Installation took 91 seconds
Installation complete
Initializing...
Setting host.docker.internal to 192.168.1.14 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 192.168.1.14 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
Setting host.containerhelper.internal to 172.17.112.1 in container hosts file
Starting Container
Hostname is cronus16
PublicDnsName is cronus16
Using NavUserPassword Authentication
Creating Self Signed Certificate
Self Signed Certificate Thumbprint 657A2D7823026760359A42F0A15A21CEFEB7595C
Modifying Service Tier Config File with Instance Specific Settings
Starting Service Tier
Registering event sources
Creating DotNetCore Web Server Instance
Using license file 'c:\run\my\license.flf'
Import License
Creating http download site
Setting SA Password and enabling SA
Creating user1 as SQL User and add to sysadmin
Creating SUPER user
Container IP Address: 172.17.114.133
Container Hostname  : cronus16
Container Dns Name  : cronus16
Web Client          : http://cronus16/BC/
Dev. Server         : http://cronus16
Dev. ServerInstance : BC
Setting cronus16 to 172.17.114.133 in host hosts file

Files:
http://cronus16:8080/ALLanguage.vsix

Container Total Physical Memory is 63.9Gb
Container Free Physical Memory is 49.8Gb

Initialization took 54 seconds
Ready for connections!
Reading CustomSettings.config from cronus16
ScriptHalted
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\2.0.16\ContainerHandling\Invoke-ScriptInNavContainer.ps1:112 char:17
+                 throw
+                 ~~~~~
    + CategoryInfo          : OperationStopped: (:) [], RuntimeException
    + FullyQualifiedErrorId : ScriptHalted
```

**Screenshots**
**error**
![image](https://user-images.githubusercontent.com/5685997/137140582-7687474f-58bc-4038-bb09-f98c9f8f073e.png)

**windows 11 pro version**
![image](https://user-images.githubusercontent.com/5685997/137141570-dcb59902-0471-4eae-b422-f9e223cddd9b.png)

bccontainerhelper and docker version
![image](https://user-images.githubusercontent.com/5685997/137141060-ee4f552d-98ee-4b88-9675-2428586b8488.png)

**docker desktop version**
![image](https://user-images.githubusercontent.com/5685997/137141343-6a5d137f-bc17-49de-b2d9-1fdb9f6d7851.png)

**Additional context**
- I ran it once after I deleted all images and containers and ran Flush-ContainerHelperCache, then I ran it again to see if it was just a first time run type of issue - the output above is from the second run, the error appeared in the same spot of execution both times.
- This machine has basically received a clean installation. I cleared all images and containers, uninstalled docker, then upgraded to windows 11, reinstalled the latest version of docker, updated bccontainerhelper to latest and ran this script.

**Comment by kevindstanley1988** [comment]
Hi Freddy,

Always running as Admin.

I have tried in the various ways to run powershell, first from VSCode terminal, which is what I have been doing for years now. Here is that version of powershell. 2021.10.0
![image](https://user-images.githubusercontent.com/5685997/137148940-3ac2b330-281f-40e9-8d1d-72e00e40e76d.png)

I also tried in the regular powershell ISE that comes windows and got the same error, here is that version. 5.1.22000.65
![image](https://user-images.githubusercontent.com/5685997/137149106-fae2674c-e726-4224-ba31-349aeb73868a.png)

I tried the docker command - it seems to hang on that with some special characters I am not sure what to make of. 
`docker exec -it cronus16 cmd`

![image](https://user-images.githubusercontent.com/5685997/137149559-35c914ad-de41-41bd-b173-0d2dfbaa346f.png)
